### PR TITLE
using media queries to improve styling across all devices

### PR DESCRIPTION
**Pull request by Laura-TM**
image

### CODE REVIEW

File: client/src/components/SingleChannelChart.js [hunk 1]
```diff
@@ -28,14 +30,19 @@ const SingleChannelChart = (props) => {
...
+				fontSize: 10,
+			},
 			display: false,
 		},
 		type: "bar",
 	};
 	return (
-		<div className="chart">
+		<div className="singleChannelChart">
 			<Bar data={state.data} options={options} />
```
Comment (askinekincihf) [inline client/src/components/SingleChannelChart.js:46]: Laura, thanks for renaming the class names to make them meaningful.

File: client/src/pages/LoginForm.css [hunk 2]
```diff
@@ -73,3 +66,33 @@ button[type="submit"] {
...
+	}
+
+	.loginButton {
+		width: 70%;
+		font-size: 20px;
+	}
+}
+
+@media (min-width: 600px) {
+	.login-group {
```
Comment (askinekincihf) [inline client/src/pages/LoginForm.css]: I am just curious why you created  `@media (min-width: 600px)` twice.

**Review by askinekincihf — APPROVED** [review]
Thanks Laura for restyling the pages.

File: client/src/pages/LoginForm.css (hunk 2 above)
Comment (Laura-TM) [reply client/src/pages/LoginForm.css]: Thanks for catching that error, I will fix it before merging :-)

**Pull request merged**